### PR TITLE
Add WT.Social https://wt.social

### PR DESCRIPTION
**Pull request by Tobi823**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -8299,6 +8299,16 @@
         ]
     },
 
+    {
+        "name": "WT.Social",
+        "url": "https://wt.social/myaccount/deactivate-account",
+        "difficulty": "easy",
```
Comment (tupaschoal) [inline _data/sites.json]:
```suggestion
        "difficulty": "impossible",
```
If data cannot be fully deleted, we deem it impossible to delete the account

File: _data/sites.json [hunk 2]
```diff
@@ -8299,6 +8299,16 @@
...
+        "notes": "Only deactivation is possible",
+        "domains": [
+            "wt.social"
+        ]
+    },
```
Comment (tupaschoal) [inline _data/sites.json]: The whole entry should be placed after `workupload` and before `Wunderlist` to keep the file sorted.

Comment (Tobi823) [reply _data/sites.json]: Sry - I thought "popular" was the name of the next entry

File: _data/sites.json (hunk 1 above)
Comment (Tobi823) [reply _data/sites.json]: ok

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**